### DISCUSSION TITLE
ja/haskell/strings/interpolating-a-string/

### DISCUSSION THREAD

**Discussion by giscus** [dogweather/forkful#43] [Page Discussions]
# ja/haskell/strings/interpolating-a-string/

(方法) Haskellでは文字列補間は言語の基本機能ではない。しかし、QuasiQuotes構文を使用するライブラリ、例えばNeatInterpolationが使い道を拡げる。歴史的には、他の言語の影響を受けている。代替手段としては、printfや++を使うことがあるが、QuasiQuotesを使った方法はより直感的で読みやすい。Haskellの型の安全性に影響を与えずに、動的な文字列を扱えるのが魅力。

https://forkful.ai/ja/haskell/strings/interpolating-a-string/

**Comment by cbh34680** [comment]
toString では動作しなかったので、Text.unpack に変更して確認できました。
(GHC のバージョンの違い ?)
